### ISSUE TITLE
Export/Import Tab or Workspace

### ISSUE THREAD

**Issue by reform-chipper** [wavetermdev/waveterm#418]
Allow exporting a single tab (or perhaps a workspace) into a proprietary file that can be later opened with Wave and executed. Export to something like a PDF or Markdown would also be cool, but is secondary. 

The broader goal is to have something like a Jupyter Notebook for shell commands.

Simple example of this vision:

![image](https://github.com/wavetermdev/waveterm/assets/162800380/896ac0ab-18d1-4a20-b207-d213a340eea1)

**Comment by sawka** [comment]
Yes, this is on the roadmap.  I'm experimenting with exporting to/from markdown (we just need to figure out how to deal with the proprietary data).

Quick question -- for your workbooks, would you want to export the output or not export the output (or having a setting to control that)?

**Comment by reform-chipper** [comment]
I think [Runme](https://docs.runme.dev) notebooks are a modified markdown, could be a start. They have other issues which prevent me from benefiting from them, but the format itself looks nice.
